Release 2.2 (#303)
BlueBrain/HighFive: CHANGELOG.md

## Version 2.2 - 2020/03/11
### New Features:
    - Compound Types: API to register and read/write structs (#78). Thanks to Richard Shaw.
    - Fixed-length strings. API via char[] and `FixedLenStringArray`(#277)
    - Enum data types (#297)
    - Datasets of HDF5 References. Support to dereference groups and datasets (#306)
    - Objects (hard/soft link) can now be deleted with `unlink` (#284). Thanks to Tom Vander Aa.
    - Attributes can be deleted with `deleteAttribute` (#239)

### Improvements:
    - Support for array of `Eigen::Matrix` (#258)
    - `Attribute`s (metadata) now support additional types (#298)
    - Add `HIGHFIVE_REGISTER_TYPE` to help registering compound types and enums (#297)
    - H5Easy: Generalized implementation for compat with `Eigen::ref` and `Eigen::Map` (#291, #293)
    - Strict compatibility with CMake 3.1 and C++11 (#304)
    - CMake: Dependencies may be re-detected on FindPackage, including for Eigen and Xtensor (#255, #304)
    - Selection: `ElementSet` working for N-dimensions (#247)

### Bug Fixes:
    - Shortcut syntax with c arrays (#273)

>     - Compat with in MSVC (Exception messages #263 and avoid throwing in `exist` check #308)

Reviewer: ```suggestion
    - Compatibility with in MSVC (Exception messages #263 and avoid throwing in `exist` check #308)
```

## Version 2.2 - 2020/03/XX
### New Features:
    - Compound: Support of Compound type (#78). Thanks to Richard Shaw.
    - Support of Fixed-length strings (#277)
    - Support of enums (#297)

### Improvements:
    - Support for array of type Eigen::Matrix (#258)
    - An attribute can be deleted with deleteAttribute (#239)

>     - A link can now be unlink (#284). Thanks to Tom Vander Aa.

Reviewer 2: ```suggestion
    - An object (hard/soft link) can now be deleted with `unlink` (#284). Thanks to Tom Vander Aa.
```

    - An annotation can now be any DataType supported by HighFive (#298)
    - Add HIGHFIVE_REGISTER_TYPE to help register of compound types and enums (#297)

> 

Reviewer 2: ```suggestion
    - H5Easy: Generalized implementation for compat with Eigen::ref and Eigen::Map (#291, #293)
    - Strict compatibility with CMake 3.1 and C++11 (#304)
    - CMake: Dependencies may be re-detected on FindPackage, including for Eigen and Xtensor (#255, #304)
    - ElementSet selection work in N-dimensions (#247)
    - 
```

> ### Bug Fixes:

Reviewer 2: ```suggestion
### Bug Fixes:
    - Shortcut syntax with c arrays (#273)
    - Exception with error messages in MSVC (#263)
```

    - Attributes can be deleted with `deleteAttribute` (#239)

### Improvements:
    - `Attribute`s (metadata) now support additional types (#298)

>     - H5Easy: Reworked for compat with `Eigen::ref` and `Eigen::Map` (#291, #293)

Reviewer: ```suggestion
    - H5Easy: Reworked for compatibility with `Eigen::ref` and `Eigen::Map` (#291, #293)
```
For consistency :)
